components/create_comment/create_comment.tsx

```diff
@@ -1285,7 +1285,7 @@ class CreateComment extends React.PureComponent<Props, State> {
         if (readOnlyChannel) {
             createMessage = Utils.localizeMessage('create_post.read_only', 'This channel is read-only. Only members with permission can post here.');
         } else {
-            createMessage = Utils.localizeMessage('create_comment.addComment', 'Add a comment...');
+            createMessage = Utils.localizeMessage('create_comment.addComment', 'Reply to this conversation...');
```

Comment:
@cwarnermm is there a reason we're saying 'Reply to this conversation…' instead of 'Reply to this thread…'? I'm 0/5 on this, but just wondering if we should be consistent with the terminology here since it is called a 'thread' in the header and the sidebar

Reply:
Agree. 0/5 but we could also simplify to "Reply..."